### DISCUSSION TITLE
Webots controller error

### DISCUSSION THREAD

**Discussion by EJ2KDJ** [cyberbotics/webots#4338] [Q&A]
Webots controller error

I cannot assign a controller to a robot. I got an error when I was trying to run a script to make the robot move forward. Then, I tried to recreate the error but I could not. And now, I’m unable to assign a controller to the robot so I'm stuck and can’t do anything.

I’m working on this tutorial made by Kajal Gada: Webots tutorial 3: Controller code to drive a differential drive robot // Webots Tutorial Python (https://www.youtube.com/watch?v=CDOrTKQAOqs&t=364s)

Is there something I can do to fix this, or do I have to start from scratch?

Thanks

Computer Specs:  Windows 10 
Graphics card: Nvidia GeForce MX330

**Comment by omichel** [comment]
I am not sure what is the problem. So, I would recommend you to restart from scratch. If you better identify the problem, please post back a new message here.
